### DISCUSSION TITLE
Missing `$namespace` in method signature

### DISCUSSION THREAD

**Discussion by gander** [spatie/laravel-translation-loader#136] [Bugs]
Interface method `\Spatie\TranslationLoader\TranslationLoaders\TranslationLoader::loadTranslations` signature: 
```
public function loadTranslations(string $locale, string $group): array
```
Usage in `\Spatie\TranslationLoader\TranslationLoaderManager::getTranslationsForTranslationLoaders`:

```
$translationLoader->loadTranslations($locale, $group, $namespace)
```

**Comment by freekmurze** [comment]
According to our tests, this should be working correctly. Feel free to PR improvements.
